ouroboros-consensus/src/unstable-tutorials/Ouroboros/Consensus/Tutorial/Simple.lhs

```diff
@@ -130,34 +130,34 @@ Next, we instantiate the `ConsensusProtocol` for `SP`:
 
 > instance ConsensusProtocol SP where
 >   type SelectView    SP = BlockNo
->
+
```

Comment:
Please don't remove these bird feet. I think the intention is for the rendered file to have blank lines in it instead of multiple separate listings.

Or am I overlooking some `lhs` idiom?

Reply:
Emacs did it for me. I fixed it thanks